src/coreclr/gc/gcpriv.h

```diff
@@ -3088,7 +3088,7 @@ class gc_heap
 
         void print()
         {
-            dprintf (3, ("last plug: %Ix, last plug reloc: %Ix, before last: %Ix, b: %Ix",
+            dprintf (3, ("last plug: %p, last plug reloc: %zu, before last: %p, b: %zu",
```

Comment:
mono uses `Iu` on win 64-bit and `u` on win 32-bit against `zu` on unix. Should we align it?
https://github.com/dotnet/runtime/blob/e6700ea21bbb17ca5058801259bc2d05957814d4/src/mono/CMakeLists.txt#L768

Reply:
This was merely to avoid the build issues and remove the suppression. We've loads of other places that need to be updated. This needs to be folded into that effort.